shell/platform/windows/platform_handler.cc

```diff
@@ -18,11 +20,123 @@ static constexpr char kSetClipboardDataMethod[] = "Clipboard.setData";
 static constexpr char kTextPlainFormat[] = "text/plain";
 static constexpr char kTextKey[] = "text";
 
-static constexpr char kUnknownClipboardFormatError[] =
-    "Unknown clipboard format error";
+static constexpr char kClipboardError[] = "Clipboard error";
+static constexpr char kUnknownClipboardFormatMessage[] =
+    "Unknown clipboard format";
 
 namespace flutter {
 
+namespace {
+
+// A scoped wrapper for GlobalLock/GlobalUnlock.
+class ScopedGlobalLock {
+ public:
+  // Attempts to acquire a global lock on |memory| for the life of this object.
+  ScopedGlobalLock(HGLOBAL memory) {
+    source_ = memory;
+    locked_memory_ = ::GlobalLock(memory);
```

Comment:
In case of `NULL`, log `GetLastError` for debugging. A helper for this may be a good idea.

Reply:
Ditto for GlobalUnlock.